app/views/partials/_singlePostCard.html.erb

```diff
@@ -2,7 +2,7 @@
   <div class="px-5">
     <div class="flex items-center space-between">
       <p class="text-md font-bold">Post #<%= post.id %> by username <%= post.author_id %></p>
-      <p class="text-sm font-bold">Comments: x, Likes: y</p>
+      <p class="text-sm font-bold">Comments: <%= post.comments_counter || 0 %>, Likes: <%= post.likes_counter || 0 %></p>
     </div>
     <p class="text-lg">
       Lorem ipsum dolor sit amet consectetur adipisicing elit. Maxime mollitia,
```

Comment:
- [ ] When I click on a specific post I am getting this paragraph but instead, I should see a post and all the comments related to it
![rl](https://user-images.githubusercontent.com/44745855/150364547-b4fb8393-fa08-422b-946a-5568f0387d5a.png)

Reply:
- [ ] I suggest that you add data in your `seed` file instead of adding these in your views it will make your codes clean